moon/controller.py

```diff
@@ -134,18 +150,18 @@ def set_cam_angle(self, angle):
         self.camera_angle = angle
         print (self.time, "app: Camera angle set to: %f" % angle)
         self.camera_change_triggered_time = self.time
-        
+
     def set_brakes(self, on):
         assert type(on) is bool, on
         self.brakes_on = on
         self.send_request('set_brakes %s\n' % ('on' if on else 'off'))
         print (self.time, "app: Brakes set to: %s" % on)
 
-    def on_driving_recovery(self, timestamp, data):
+    def on_driving_recovery(self, data):
```

Comment:
this is a bit crazy - we should decide/freeze what `on_something` should receive ... 3rd option is only `self`

Reply:
It seemed to me the move was towards unifying around the handler(getattr...) approach (some even said to make it part of node.py). At least the handler was used in all the other nodes outside the app. I was keeping it as it was as long as I could but eventually it was too much to be adding every new message type twice - once the actual `on_something` function in one file and then again in different file as `if channel=="something"`